projects/packages/jetpack-mu-wpcom/src/features/wpcom-admin-bar/wpcom-admin-bar.scss

```diff
@@ -92,4 +92,50 @@
 	float: none;
 	position: absolute;
 	right: 0;
+
+	@media (max-width: 782px) {
+		background-color: blue;
```

Comment:
> blue

This is for testing only for now, right?

Reply:
only for testing!
I converted this PR to a draft...🔄